polyfill/ci_codecov_test262.sh

```diff
@@ -7,6 +7,31 @@ if [ -n "$HEAD_SHA" ]; then
   SHA_ARG="-c $HEAD_SHA"
 fi
 
-export COVERAGE=yes
-npm run test262
-codecov $SHA_ARG -F test262 -f coverage/test262.lcov
+npm run build262
+
+export NODE_V8_COVERAGE=coverage/tmp/
+rm -rf $NODE_V8_COVERAGE
+
+# Run the tests in chunks
+# Works around memory leak with vm.Script and NODE_V8_COVERAGE
+node test262-harness.mjs "*.js"
+node test262-harness.mjs "Calendar/**"
+node test262-harness.mjs "Duration/**"
+node test262-harness.mjs "Instant/**"
+node test262-harness.mjs "Now/**"
+node test262-harness.mjs "PlainDate/**"
+node test262-harness.mjs "PlainDateTime/**"
+node test262-harness.mjs "PlainMonthDay/**"
+node test262-harness.mjs "PlainTime/**"
+node test262-harness.mjs "PlainYearMonth/**"
+node test262-harness.mjs "TimeZone/**"
+node test262-harness.mjs "toStringTag/**"
+node test262-harness.mjs "ZonedDateTime/**"
```

Comment:
Is this list intentionally omitting Regex, UserCalendar, and UserTimezone? Also, it's just hard in general to identify what if anything is missing.
```suggestion
node test262-harness.mjs "*.js"
for testSubdir in $(find test262/test/*/Temporal/* -maxdepth 0 -type d -exec basename '{}' ';' | sort -u); do
  node test262-harness.mjs "$testSubdir/**"
done
```